utils/material_plots_2D.py

```diff
@@ -1,80 +1,167 @@
+"""
+This script must be called with python: 'python material_scan_2D.py --{argument} {value}'.
+The output files are saved in data/{outputDir}/name.suffix.
+If no outputDir is specified, it will be data/plots/name.suffix.
+"""
+
 from __future__ import print_function
+
 import argparse
 import math
+import sys
+from os import fspath
+from os.path import expandvars
+from pathlib import Path
 
-import sys,os
-sys.path.append(os.path.expandvars("$FCCSW") + "/Examples/scripts")
-from plotstyle import FCCStyle
 import ROOT
 
+sys.path.append(expandvars("$FCCSW") + "/Examples/scripts")
+from plotstyle import FCCStyle
+
 
 def main():
-    parser = argparse.ArgumentParser(description='Material Plotter')
-    parser.add_argument('--fname', "-f", dest='fname', type=str, help="name of file to read")
-    parser.add_argument('--angleMin', dest='angleMin', default=6, type=float, help="minimum eta/theta/cosTheta")
-    parser.add_argument('--angleMax', dest='angleMax', default=6, type=float, help="maximum eta/theta/cosTheta")
-    parser.add_argument('--angleDef', dest='angleDef', default="eta", type=str, help="angle definition to use: eta, theta or cosTheta, default: eta")
-    parser.add_argument('--angleBinning', "-b", dest='angleBinning', default=0.05, type=float, help="eta/theta/cosTheta bin width")
-    parser.add_argument('--nPhiBins', dest='nPhiBins', default=100, type=int, help="number of bins in phi")
-    parser.add_argument('--x0max', "-x", dest='x0max', default=0.0, type=float, help="Max of x0")                                                                                                                                                
+    parser = argparse.ArgumentParser(description="Material Plotter")
+    parser.add_argument(
+        "--inputFile", "-f", type=str, help="relative path to the input file"
+    )
+    parser.add_argument(
+        "--angleMin",
+        default=6,
+        type=float,
+        help="Minimum eta/theta/cosTheta",
+    )
+    parser.add_argument(
+        "--angleMax",
+        default=6,
+        type=float,
+        help="Maximum eta/theta/cosTheta",
+    )
+    parser.add_argument(
+        "--angleDef",
+        default="eta",
+        choices=["eta", "theta", "cosTheta", "thetaRad"],
+        type=str,
+        help="Angle definition to use: eta, theta, thetaRad or cosTheta; default: eta",
+    )
+    parser.add_argument(
+        "--angleBinning",
+        "-b",
+        default=0.05,
+        type=float,
+        help="Eta/theta/cosTheta bin width",
+    )
+    parser.add_argument(
+        "--nPhiBins",
+        default=100,
+        type=int,
+        help="Number of bins in phi",
+    )
+    parser.add_argument("--x0Max", "-x", default=0.0, type=float, help="Max of x0")
```

Comment:
```suggestion
    parser.add_argument("--x0max", "-x", default=0.0, type=float, help="Max of x0")
```
Can we keep the original capitalization? There is no real need to change it as far as I can see.

Reply:
I thought of it to use camelCase consistently. Then I do not have to remember which arguments do and which do not

Reply:
If you really want to, add an alternative flag, but see the [other comment.](https://github.com/key4hep/k4geo/pull/392/files#r1776702228)